dune_api_scripts/tests/test_utils.py

```diff
@@ -1,7 +1,7 @@
 import time
 import unittest
 
-from ..utils import ensure_that_download_is_recent, dune_address
+from ..utils import ensure_that_download_is_recent, hex2bytea
```

Comment:
Renamed methods because its no longer being used only for addresses, but also transaction hashes.